operator/config/crd/bases/dragonflydb.io_dragonflydbs.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
This is autogenerated based on the `dragonflydb_types.go` which is the main resources to change the schema of the `DragonFlyDb` resource.